styles/widgets/android5/treeView.android5.less

```diff
@@ -34,6 +34,10 @@
 }
 
 .dx-treeview {
+    .dx-treeview-search {
+        margin-bottom: -3px;
```

Comment:
This negative margin looks strange. Can we refactor the less to prevent negative margin usage?

Reply:
Changed